pkg/operator/tpr.go

```diff
@@ -59,7 +59,7 @@ func (o *Operator) createTPR() error {
 		}
 	}
 
-	return err
```

Comment:
@travisn but if TPR fails to create for any reason, shouldnt we raise that? Without TPR, the cluster will not be created